remo/domain/dataset.py

```diff
@@ -177,49 +147,36 @@ def export_annotations_to_file(
         Exports annotations in given format
 
         Args:
-            annotation_set_id: annotation set id
             output_file: output file to save
+            annotation_set_id: annotation set id
             annotation_format: can be one of ['json', 'coco', 'csv'], default='json'
             full_path: uses full image path (e.g. local path), can be one of ['true', 'false'], default='false'
             export_coordinates: converts output values to percentage or pixels, can be one of ['pixel', 'percent'], default='pixel'
         """
-        self.sdk.export_annotations_to_file(
-            annotation_set_id,
-            output_file,
-            annotation_format=annotation_format,
-            full_path=full_path,
-            export_coordinates=export_coordinates,
-        )
+        annotation_set = self.get_annotation_set_or_default(annotation_set_id)
+        if annotation_set:
+            self.sdk.export_annotations_to_file(
```

Comment:
@vmanilo shouldn't this be a method of annotation set rather than SDK?

in `export_annotations` we use the annotation set's method